gp_predict.f95

```diff
@@ -1534,24 +1569,24 @@ subroutine gpFull_covarianceMatrix_sparse(this,error)
          this%map_yPrime_globalY(i_yPrime) = i_globalY
       enddo
 
-      call reallocate(this%covariance_subY_y, this%n_globalSparseX, n_globalY, zero = .true.)
-      call reallocate(this%covariance_subY_subY, this%n_globalSparseX, this%n_globalSparseX, zero = .true.)
-      call reallocate(this%covarianceDiag_y_y, n_globalY, zero = .true.)
-      call reallocate(this%lambda, n_globalY, zero = .true.)
-      call reallocate(this%inverse_sparse_full, this%n_globalSparseX, n_globalY, zero = .true.)
+      call reallocate(this%covariance_subY_y, this%n_globalSparseX, n_globalY, zero=.true.)
+      call reallocate(this%covariance_subY_subY, this%n_globalSparseX, this%n_globalSparseX, zero=.true.)
+      call reallocate(this%covarianceDiag_y_y, n_globalY, zero=.true.)
+      call reallocate(this%lambda, n_globalY, zero=.true.)
+      call reallocate(this%inverse_sparse_full, this%n_globalSparseX, n_globalY, zero=.true.)
 
-      allocate( covariance_subY_currentX_y(n_globalY),covariance_subY_currentX_suby(this%n_globalSparseX) )	
-      covariance_subY_currentX_y = 0.0_dp	
+      allocate( covariance_subY_currentX_y(n_globalY),covariance_subY_currentX_suby(this%n_globalSparseX) )
+      covariance_subY_currentX_y = 0.0_dp
       covariance_subY_currentX_suby = 0.0_dp
 
-      do i_coordinate = 1, this%n_coordinate
+      coordinate_loop: do i_coordinate = 1, this%n_coordinate
```

Comment:
loop labels don't seem to be not needed, but I suppose they only improve clarity

Reply:
Left over from debugging. That loop is quite long. Can be removed.

Reply:
Doesn't do any harm, let's keep it.